roles/config-docker-compose/README.md

```diff
@@ -0,0 +1,56 @@
+config-docker-compose
+=====================
+
+The config-docker-compose role is associated with installing the docker compose packages.
+
+Requirements
+------------
+
+N/A
+
+
+Role Variables
+--------------
+
+Defaults file has example of variables that can be used.
+
+| Variable | Description | Required | Defaults |
+|:--------:|:-----------:|:--------:|:--------:|
+|**docker_compose_packages**|  Packages required for docker compose. | yes | in defaults file |
+|**docker_install**| Should docker be installed | true | false |
+|**docker_compose_install**| Should docker compose be installed |true | false |
+
+
+Dependencies
+------------
+config-docker role is required.
+
+Example Playbooks
+----------------
+
+``` yaml
+- hosts: bastion
+  roles:
+    - role: config-docker-compose
+```
+
+Example Inventory
+----------------
+
+```
+docker_install: true
+docker_compose_install: true
+docker_username: bob
```

Comment:
Make sure to call out that `docker_username` is use by the `config-docker` role - i.e.: not **this** role.

Reply:
Added a comment that this variable is from the docker role.